pygfx/objects/_base.py

```diff
@@ -176,10 +176,18 @@ def remove(self, *objects):
                 pass
         return self
 
-    def traverse(self, callback):
-        callback(self)
-        for child in self.children:
-            child.traverse(callback)
+    def traverse(self, callback, include_non_visible=False):
```

Comment:
I expect that this function will be used to implement update logic in game loops and animations. So it seems like a surprise/gotcha/bad idea to filter out invisible things by default.

Can you flip that around?